back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: batch/v1
+kind: Job
```

Comment:
this way it runs the script to completion, or tries to. if something fails then it won't strrt over and waste cpu